app/views/orders/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_for(order) do |f| %>
+  <% if order.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(order.errors.count, "error") %> prohibited this order from being saved:</h2>
+
+      <ul>
+      <% order.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :product_name %>
+    <%= f.text_field :product_name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :product_count %>
+    <%= f.text_field :product_count %>
+  </div>
+
+  <div class="field">
+    <%= f.select :customer_id, 
+    options_for_select(
+        Customer.all.map do |customer| 
+        #HERE 42
+        end
+    ) %>
```

Comment:
This is an extra select for customers but there is another collection select below, this code should be removed if it is not being used.